scenarios/Bruteforce_CWE-307/exploit/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # partly taken from phocean/msf on https://hub.docker.com/r/phocean/msf/ or https://github.com/phocean/dockerfile-msf
 
-FROM ubuntu:bionic
+FROM ubuntu:bionic-20210702
 
 ARG DEBIAN_FRONTEND=noninteractive
 
```

Comment:
2021-08-09 16:37:45,032 - INFO - [NORMAL] 61c4e641929a3857 - GET: https://090138215ad4513a/shop.html
2021-08-09 16:37:45,032 - INFO - [NORMAL] 61c4e641929a3857 - GET: https://090138215ad4513a/about.html
2021-08-09 16:37:45,657 - INFO - [ATTACKER] 09adf75734921684 - Executing the exploit step attack now at 1628519865.6570263
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/home/tk/.local/lib/python3.7/site-packages/docker/api/client.py", line 259, in _raise_for_status
    response.raise_for_status()
  File "/home/tk/.local/lib/python3.7/site-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: http+docker://localhost/v1.41/containers/5a65972e65de025acd07c7a0674f0fa163ca69bd4c5ca212fe2967118dee1e38/attach?stdin=1&stream=1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.7/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/tk/Documents/LID-DS/lid_ds/core/objects/attacker.py", line 38, in execute_exploit_at_time
    socket = self.container.attach_socket(params={'stdin': 1, 'stream': 1})
  File "/home/tk/.local/lib/python3.7/site-packages/docker/models/containers.py", line 112, in attach_socket
    return self.client.api.attach_socket(self.id, **kwargs)
  File "/home/tk/.local/lib/python3.7/site-packages/docker/utils/decorators.py", line 19, in wrapped
    return f(self, resource_id, *args, **kwargs)
  File "/home/tk/.local/lib/python3.7/site-packages/docker/api/container.py", line 111, in attach_socket
    headers=headers
  File "/home/tk/.local/lib/python3.7/site-packages/docker/api/client.py", line 309, in _get_raw_response_socket
    self._raise_for_status(response)
  File "/home/tk/.local/lib/python3.7/site-packages/docker/api/client.py", line 261, in _raise_for_status
    raise create_api_error_from_http_exception(e)
  File "/home/tk/.local/lib/python3.7/site-packages/docker/errors.py", line 31, in create_api_error_from_http_exception
    raise cls(e, response=response, explanation=explanation)
docker.errors.NotFound: 404 Client Error: Not Found ("b'No such container: 5a65972e65de025acd07c7a0674f0fa163ca69bd4c5ca212fe2967118dee1e38'")

2021-08-09 16:37:45,900 - INFO - [NORMAL] 6d87853e7f5b76f7 - GET: https://090138215ad4513a/blog.html
2021-08-09 16:37:45,900 - INFO - [NORMAL] 6d87853e7f5b76f7 - POST: user user3 file /tmp/tmpyj7daiv7
2021-08-09 16:37:45,900 - INFO - [NORMAL] 6d87853e7f5b76f7 - GET: https://090138215ad4513a/about.html

Reply:
Attacker shuts down at beginning.